src/lib/analytics/analytics.js

```diff
@@ -1,44 +1,47 @@
 //@flow
-import logger from '../../lib/logger/pino-logger'
 import Config from '../../config/config'
 
 let Amplitude, FS, Rollbar
-const log = logger.child({ from: 'analytics' })
 
-export const initAnalytics = async (goodWallet: GoodWallet, userStorage: UserStorage) => {
+let log = console
+
+export const initAnalytics = async (goodWallet: GoodWallet, userStorage: UserStorage, logger) => {
+  log = logger.child({ from: 'analytics' })
   const identifier = goodWallet.getAccountForType('login')
-  const emailOrId = (await userStorage.getProfileFieldValue('email')) || identifier
-  if (global.Rollbar && Config.rollbarKey) {
-    Rollbar = global.Rollbar
-    global.Rollbar.configure({
-      accessToken: Config.rollbarKey,
-      captureUncaught: true,
-      captureUnhandledRejections: true,
-      payload: {
-        environment: Config.env,
-        person: {
-          id: emailOrId,
-          identifier,
+  if (identifier) {
```

Comment:
why? identifier should always be available when analytics are initialized.
since we dont call initanalytics again this will means there are cases that analytics isnt initialized at